File: SDKs/googleapis.com/adexchangebuyer/v1.3/python/src/sdk/models/shared/billinginfo.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class BillingInfo:
    r"""BillingInfo
    The configuration data for an Ad Exchange billing info.
    """
    
    account_id: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('accountId') }})
    account_name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('accountName') }})
    billing_id: Optional[list[str]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('billingId') }})
    kind: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('kind') }})
    

File: SDKs/googleapis.com/adexchangebuyer/v1.3/java/lib/src/main/java/openapisdk/models/shared/BillingInfo.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * BillingInfo
 * The configuration data for an Ad Exchange billing info.
**/
public class BillingInfo {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("accountId")
    public Integer accountId;
    public BillingInfo withAccountId(Integer accountId) {
        this.accountId = accountId;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("accountName")
    public String accountName;
    public BillingInfo withAccountName(String accountName) {
        this.accountName = accountName;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("billingId")
    public String[] billingId;
    public BillingInfo withBillingId(String[] billingId) {
        this.billingId = billingId;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("kind")
    public String kind;
    public BillingInfo withKind(String kind) {
        this.kind = kind;
        return this;
    }
}